iterators.tex

```diff
@@ -608,7 +609,7 @@
 \tcode{Iterator}~(\ref{iterators.iterator}). It
 defines requirements for a type whose referenced values can be read (from the requirement for
 \tcode{Readable}~(\ref{iterators.readable})) and which can be both pre- and post-incremented. However,
-input iterators are not required to be comparable for equality.
+input iterators are not required to satisfy \tcode{EqualityComparable}~(\ref{concepts.lib.compare.equalitycomparable}).
```

Comment:
That raises the issue in my mind of what `EqualityComparable` means in ISO/IEC 14882, and how it is related to what `EqualityComparable` means in this document, but maybe that's OK. Also, I think this should be a note, since the lack of the `EqualityComparable` requirement is already specified in the `InputIterator` concept definition.